packages/node-core/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.5.0] - 2023-06-15
```

Comment:
Can you please update the links at the bottom. We suggest using chan for future.

Same for other packages